internal/drive_upload.Rmd

```diff
@@ -0,0 +1,129 @@
+---
+title: "Playing with `drive_upload()`"
+author: "Lucy D’Agostino McGowan"
+date: "7/14/2017"
+output: github_document
+---
+
+```{r}
+library("googledrive")
+library("testthat")
```

Comment:
Not sure of your motivation but you can also set `error = TRUE` globally or chunk wise to render in the presence of errors and reveal them.